boot/zephyr/sample.yaml

```diff
@@ -5,6 +5,7 @@ sample:
 tests:
   sample.bootloader.mcuboot:
     tags: bootloader_mcuboot
+    platform_allow:  nrf52840dk_nrf52840 frdm_k64f disco_l475_iot1
```

Comment:
Does `twister` always require duplicated information on `platform_allow` and `integration_platforms`?

Reply:
no, if it is called with --integration for instance.